docs/content/dagster-cloud/managing-deployments/alerts.mdx

```diff
@@ -0,0 +1,225 @@
+
+5. In the **Create alert policy** window, select an [alert type](#alert-policy-types) and enter a name and description for the alert.
+
+6. **If creating an asset or run alert**, fill in the following:
+
+   <details>
+     <summary>
+       <strong>Asset alert policies</strong>
+     </summary>
+     <ul>
+       <li>
+         <strong>Target</strong> - Select the assets that the alert targets:
+         <ul>
+           <li>
+             <strong>All assets</strong> - All assets in the deployment
+           </li>
+           <li>
+             <strong>Group</strong> - All assets in a specific asset group in
+             the deployment
+           </li>
+           <li>
+             <strong>Asset key</strong> - A specific asset key in the deployment
+           </li>
+         </ul>
+       </li>
+       <li>
+         <strong>Events</strong> - Select the event(s) that should trigger the
+         alert. For example: asset materialization failure, asset check passed.
+       </li>
+     </ul>
+   </details>
+
+   <details>
+     <summary>
+       <strong>Run alert policies</strong>
+     </summary>
+     <ul>
+       <li>
+         <strong>Tags</strong> - Add tag(s) for the alert policy. Jobs with
+         these tags will trigger the alert. For example:{" "}
+         <code>level:critical</code> or <code>team:sales</code>
+       </li>
+       <li>
+         <strong>Events</strong> - Select whether the alert should trigger on
+         job success, failure, or both
+       </li>
+     </ul>
+   </details>
+
+7. In the **Notification service** section, select how the alert will be delivered. Refer to the [Supported notification services](#supported-notification-services) section for information about setting up these services.
+
+   **Note**: If creating an asset alert, an additional **Email asset owners (experimental)** option will be available.
+
+8. When finished, click **Save policy**.
+
+### Editing alert policies
+
+To edit an existing alert policy, click the **Edit** button next to the policy.
+
+### Enabling and disabling alert policies
+
+To enable or disable an alert, use the toggle on the left side of the alert policy.
+
+### Deleting alert policies
+
+To delete an alert policy, click the **Delete** button next to the policy. When prompted, confirm the deletion.
+
+---
+
+## Managing alert policies with the dagster-cloud CLI
+
+When creating an alert policy using the CLI, only certain `event_types` can be specified together. You can specify multiple job run-based event types together (`JOB_SUCCESS`, `JOB_FAILURE`), or a tick-based event type (`TICK_FAILURE`), but attempting to mix these will result in an error.
+
+With the [`dagster-cloud` CLI](/dagster-cloud/managing-deployments/dagster-cloud-cli), you can:
+
+- [Set a full deployment's policies](#setting-a-full-deployments-policies)
+- [View a full deployment's policies](#viewing-a-full-deployments-policies)
+
+### Setting a full deployment's policies
```

Comment:
instead of saying `full deployment` here, we should have a callout that alerting as a whole is only supported on full deployments, not branch deployments

Reply:
I wound up splitting the CLI stuff out into its own guide - I think it makes it easier to find/digest. Added this info to the intro of that guide.